### DISCUSSION TITLE
GitHub education student verification not supporting Non-English documents?

### DISCUSSION THREAD

**Discussion by buer19970329** [community/community#43801] [GitHub Education]
Hi there,

I am a student from China. Every time when I uploaded the photo of my student ID card (in Chinese) and pressed the "process my application" button, it showed my application could not be reviewed because I hadn't completed my profile. However, I have already completed my profile, and even changed my GitHub name to my real name (in Chinese and English)!

I don't know what my mistake was. Please let me know how to verify it. Thanks in advance!

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
